portal/templates/redesign/help-and-support_new.html

```diff
@@ -0,0 +1,87 @@
+{% extends 'redesign/base_new.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<div class="banner banner--teacher row">
+    <div class="col-sm-5 col-sm-offset-2">
+        <h1 class="banner__text--primary">Help and support</h1>
+        <p class="body-text">If you&rsquo;ve run into problems, please look at our <a href="#faqs">FAQs</a> below for advice to commonly asked questions. If you still can&rsquo;t
+            find the answers, please <a href="#contact">contact us</a>. One of the Code for Life team will get back to you and resolve your issues.</p>
+    </div>
+    <div class="col-sm-5">
+        <div class="hexagon"><div class="hexagon-in1"><div class="hexagon-in2 hexagon-in2--about"></div></div></div>
```

Comment:
This is repeated from another page.  If you find that happening, make an include, so if it changes, you only have one place to change it.  This is true of the whole section, really.  If only the content is changing, make a code block, and feed in the content.